src/Uno.UI/UI/Xaml/RoutedEventFlag.cs

```diff
@@ -98,8 +98,10 @@ internal static class RoutedEventFlagExtensions
 			| RoutedEventFlag.ManipulationCompleted;
 
 		private const RoutedEventFlag _isGesture = // 0b0000_0000_0001_1111___0000_0000_0000_0000___0000_0000_0000_0000___0000_0000_0000_0000
-			  RoutedEventFlag.Tapped
-			| RoutedEventFlag.DoubleTapped;
+			RoutedEventFlag.Tapped
+			| RoutedEventFlag.DoubleTapped
+			| RoutedEventFlag.RightTapped;
+			//| RoutedEventFlag.Holding;
```

Comment:
![Codacy](https://app.codacy.com/assets/images/favicon.png) Issue found: [Remove this commented out code.](https://app.codacy.com/gh/unoplatform/uno/pullRequest?prid=4826514)